benchmark/_cli.js

```diff
@@ -107,8 +106,8 @@ CLI.prototype.benchmarks = function() {
 };
 
 CLI.prototype.shouldSkip = function(scripts) {
-  const filters = this.optional.filter || [];
-  const excludes = this.optional.exclude || [];
+  const filters = this.optional.filter ? [...this.optional.filter] : [];
+  const excludes = this.optional.exclude ? [...this.optional.exclude] : [];
```

Comment:
This looks unrelated

Reply:
Emmm... I have forgot why a add spread syntax in there. This PR has been a long time coming, I have made a code review, and I think it is unnecessary too.I will restore it